astroimtools/tests/test_cutout_tool.py

```diff
@@ -0,0 +1,154 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import tempfile
+from numpy.testing import assert_allclose, assert_almost_equal
+
+import os
+import numpy as np
+from astropy.table import Table
+import astropy.units as u
+from astropy.io import fits
+from astropy.table import Table
+from astropy import wcs
+from astropy.io.fits.tests import FitsTestCase
+from astropy.coordinates import SkyCoord
+
+from ..cutout_tools import cutout_tool
+
+
+class TestCutoutTool(FitsTestCase):
+    def setup(self):
+        self.temp_dir = tempfile.mkdtemp(prefix='fits-test-')
+        fits.conf.enable_record_valued_keyword_cards = True
+        fits.conf.extension_name_case_sensitive = False
+        fits.conf.strip_header_whitespace = True
+        fits.conf.use_memmap = True
+
+    @staticmethod
+    def construct_test_image():
+        # Construct data: 10 X 10 array where
+        # the value at each pixel (x, y) is x*10 + y
+        data = np.array([i * 10 + np.arange(10) for i in range(10)])
+
+        # Construct a WCS for test image
+        # N.B: Pixel scale should be 1 deg/pix
+        w = wcs.WCS()
+        w.wcs.crpix = [5, 5]
```

Comment:
If you want ``[0, 45]`` to be at the center of the image, you'll need this to be ``[5.5, 5.5]``. This is needed for the reproject results to be sensible (otherwise things will get offset when you rotate).

Reply:
Thanks! I instead changed the matrix to a 9 by 9